packages/forge/blocks/zendeskChat/actions/authenticate.ts

```diff
@@ -0,0 +1,48 @@
+import { createAction, option } from '@typebot.io/forge'
+import { sign } from 'jsonwebtoken';
+import { auth } from '../auth'
+
+export const authenticate = createAction({
+  name: 'Authenticate User',
+  auth,
+  options: option.object({
+    userId: option.string.layout({
+      label: 'User ID',
+      isRequired: true,
+    }),
+    name: option.string.layout({
+      label: 'Name',
+      isRequired: true,
+    }),
+    email: option.string.layout({
+      label: 'Email',
+      isRequired: true,
+    }),
+    emailVerified: option.string.layout({
+      label: 'Email Is Verified'
+    }),
+    tokenVariableId: option.string.layout({
+      label: 'Token Variable',
+      inputType: 'variableDropdown',
+      isRequired: true
+    }),
+  }),
+  run: {
+    server: async ({
```

Comment:
**suggestion:** Add input validation for required fields

Consider adding more robust input validation to ensure all required fields are present and valid before attempting to sign the token. This will help prevent potential errors and improve the overall reliability of the authentication process.